node/text-generation/src/main.js

```diff
@@ -0,0 +1,29 @@
+import { HfInference } from '@huggingface/inference';
+import { getStaticFile, throwIfMissing } from './utils.js';
+
+export default async ({ req, res }) => {
+  throwIfMissing(process.env, ['OPENAI_API_KEY']);
```

Comment:
why does it need the Open AI API key? If its required, it needs to be included in the readme and the env.d.ts as well. Its missing in both the places

Reply:
Probably needs a OpenAI key for the embeddings

Reply:
Scratch that, we're not running embeddings